rust/arrow/src/csv/reader.rs

```diff
@@ -278,29 +283,51 @@ impl<R: Read> Reader<R> {
             .map(|i| {
                 let field = self.schema.field(*i);
                 match field.data_type() {
-                    &DataType::Boolean => {
-                        self.build_primitive_array::<BooleanType>(rows, i)
+                    &DataType::Boolean => self.build_primitive_array::<BooleanType>(
```

Comment:
Why we need to pass in `self.line_number` in `build_primitive_array`?

Reply:
oops, good point